packages/e2e-tests/next-app-using-experimental-output-file-tracing/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "next-app-using-serverless-trace",
```

Comment:
Can you also update all instances of `next-app-using-serverless-trace` to `next-app-using-experimental-output-file-tracing` and add the app to here https://github.com/serverless-nextjs/serverless-next.js/blob/master/.github/workflows/e2e-tests.yml#L90